### PR TITLE
COMMERCE-1980 Order emails page

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

File: commerce-order-web/src/main/resources/META-INF/resources/order/emails.jsp [hunk 1]
```diff
@@ -22,95 +22,133 @@ CommerceOrderEditDisplayContext commerceOrderEditDisplayContext = (CommerceOrder
...
+			className="com.liferay.commerce.notification.model.CommerceNotificationQueueEntry"
+			escapedModel="<%= true %>"
+			keyProperty="commerceNotificationQueueEntryId"
+			modelVar="commerceNotificationQueueEntry"
+		>
+
+			<%
+			User notificationUser = commerceOrderEditDisplayContext.getNotificationUser(commerceNotificationQueueEntry);
+
+			PortletURL rowURL = renderResponse.createRenderURL();
```
Comment (ethanbustad) [inline commerce-order-web/src/main/resources/META-INF/resources/order/emails.jsp]: Why have the `rowURL` logic up here, but the `data` logic down lower? Looks like they're used in the same place -- makes sense to me to declare them in the same scriptlet.

File: commerce-order-web/src/main/resources/META-INF/resources/email/detail.jsp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<%
+CommerceOrderEditDisplayContext commerceOrderEditDisplayContext = (CommerceOrderEditDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+CommerceNotificationQueueEntry commerceNotificationQueueEntry = commerceOrderEditDisplayContext.getCommerceNotificationQueueEntry();
+%>
+
+<div class="border-0 sheet">
+	<div class="commerce-notification-queue-entry-header">
+
```
Comment (ethanbustad) [inline commerce-order-web/src/main/resources/META-INF/resources/email/detail.jsp]: I think a couple newlines could be removed here.

File: commerce-order-web/src/main/resources/META-INF/resources/order/emails.jsp [hunk 3]
```diff
@@ -22,95 +22,133 @@ CommerceOrderEditDisplayContext commerceOrderEditDisplayContext = (CommerceOrder
...
+
+<div id="<portlet:namespace />side-panel-root"></div>
+<div id="<portlet:namespace />side-panel-wrapper"></div>
+
+<aui:script require="commerce-frontend-js/js/side_panel/entry.es as SidePanel">
+	new SidePanel.default(
+		"<portlet:namespace />sidePanel",
+		"<portlet:namespace />side-panel-root",
+		{
+			size: "lg",
```
Comment (ethanbustad) [inline commerce-order-web/src/main/resources/META-INF/resources/order/emails.jsp:124]: these fields should be sorted alphabetically

**Comment by alecsloan** [comment]
ci:test

**Review by ethanbustad — APPROVED** [review]

**Pull request closed without merging**